### ISSUE TITLE
Call for Outreachy Proposals!!!

### ISSUE THREAD

**Issue by SidharthBansal** [publiclab/plots2#3726]
We are having ``Planning for expanded community stats system`` as our Outreachy Project this winter.
Refer #3498 

Folks are requested to post the plan for the fellowship program in form of proposal.
They can brainstorm project related ideas here. 

****

### Update

We now have an official call for proposals posted at PublicLab.org, here:

https://publiclab.org/notes/warren/10-23-2018/call-for-outreachy-2018-19-proposals

****

I need to check the exact requirements in Outreachy but these have some of the kind of information that will be helpful in framing out a proposal:
https://publiclab.org/notes/warren/02-28-2017/call-for-proposals

I hope this is helpful!
A few tips for making awesome proposals:
* Please [submit the proposals at publiclab.org](https://publiclab.org/notes/warren/10-23-2018/call-for-outreachy-2018-19-proposals) first. After all the reviews you can submit it on the official website.
* Your information so that we can know you better (not mandatory !) .
* Include weekly timeline: Explain what you will be doing in which week. Divide the Outreachy period into months in bigger tasks. Then into weeks into smaller tasks.
* Include demos/images/gifs to demonstrate your idea
* Include Mock Ups, Flow Charts, Data Flow Diagrams, etc to explain the workflow in simple language. So that people in logistic department can review your work and suggest changes.
* PRs merged & Issues reported: This is the most crucial aspect of any application. We want people to be well versed with our system. Please mention all the issues reported, pr merged, suggestions etc in the proposal.
* Start early and have many revisions of the proposal to increase your chances.

These tips are general tips.
Are you tensed? No need to worry!!! :smile: We all are there to help you at each stage of proposing the proposal to the manufacturing of the end product. So just be happy and code.

Best of luck folks. :tada: Hope we will have great time this year with you all. :+1: 
Here's the one for this last year's program! :1st_place_medal: :1234: :100: 
https://publiclab.org/notes/warren/02-15-2018/call-for-summer-of-code-2018-proposals

 Thanks and more soon! :smile: :tada: 

We are participating in Outreachy for the first time. Hoping you all will have awesome experience!!!

**Comment by quinn-codes-synthesis** [comment]
What's the best way to go about this if we're not really sure what timeline to expect of ourselves? i.e. I'm generally a very fast learner, but I haven't yet had the opportunity to see how quickly I pick things up when I'm coding full-time (currently in an unrelated full-time job, coding whenever I can carve out some free time in my schedule). I'm also thinking I should build in some buffer time for things like getting stuck on an issue, debugging, research for how best to tackle a problem, etc etc.

I guess what I'm asking is -- should I be stressing about figuring out an exact schedule and sticking to it, or is this more about making a rough statement of goals and priorities broken down by week?

**Comment by SidharthBansal** [comment]
Hi Quinn,
Glad to see you.
We are looking for an expected timeline so that we can see your progress
throughout the project. With help of the timeline, we introduce modularity
into the project so that each piece of code is reviewed thoroughly and
merged.

Yes it is great idea to include testing, bug fixes, documentation into the
proposal.

No strict deadlines are needed. Just give how you want to break down the
project into different weekly tasks.

Best of luck for proposal.

**Comment by SidharthBansal** [comment]
>>rough statement of goals and priorities broken down by week?

Is OK.

**Comment by quinn-codes-synthesis** [comment]
Fantastic, thank you!

(the below is crossposted at #3498)

Woops, Github was being finicky yesterday and I ended up asking basically the same question here and in the other thread (it wasn't showing me that the post went through).

Thanks for the response! What is the best way to submit a proposal work-in-progress? Email to you and the other mentors, posting it here, something else? Through the Outreachy site directly?

**Comment by SidharthBansal** [comment]
We will have public conversation always so as to have open source environment as Outreachy is open source.
You need to create research notes on publiclab.org
Tag the mentors with their names.

**Comment by quinn-codes-synthesis** [comment]
Ah, research notes. That's the tidbit I was missing. Thank you so much!

**Comment by SidharthBansal** [comment]
Don't forget to fill the proposal on main outreach website before the
deadline.


We want to help you with your application so we are reviewing it on
publiclab research notes section.

Actual evaluation will be based on the final application ie proposal
submitted to Outreachy website. So don't forget to submit them on the
Outreachy website.

Does this makes sense?

**Comment by quinn-codes-synthesis** [comment]
Yep, perfect!

What is the appropriate method for tagging a user on Public Lab? I see the syntax `[username](/profile/username)` is mentioned [here](https://publiclab.org/wiki/authoring-help) as a method to link a user's profile -- but I thought that perhaps you meant writing `warren, bansal_sidharth2996` et al. in the tags section, so I figured I'd clarify. :)

**Comment by SidharthBansal** [comment]
Yeah you are right.
Jeff has @warren username.
You just need to list the names by seeing them on their profiles

**Comment by jywarren** [comment]
(@quinn-codes-synthesis this should help you post!)

Hi, everyone! I'm sorry there was some confusion -- since, unlike in our other Summer of Code programs, we have a single project we've submitted, we weren't sure we needed a call for proposals page. But although we'd love to see group discussion and collaboration around refining the proposals, it does seem helpful to ask each applicant to post a draft proposal (in addition to the one you'll send to Outreachy.org).

So I've posted a call for proposals here, where I'd like to ask each of you to open a draft proposal and get some comments on it! Thank you!!

https://publiclab.org/notes/warren/10-23-2018/call-for-outreachy-2018-19-proposals

**Comment by bansalsidharthcode** [comment]
University and college students who are applying are requested to provide information about their school calendar and class credits. We will ask for your current term and your next term. Please have that information before starting the process.

**Comment by SidharthBansal** [comment]
All the participants of Outreachy are requested to submit the application before** October 30 at 4pm UTC ** on the main Outreachy website. Failure to do so will result in failure from the Outreachy program.

Applicants who create an application can edit it until the project application deadline. After your project's application deadline, applicants will NOT be able to create a new application. Applicants who have submitted an application cannot edit their application, but they will be able to continue recording and editing their contributions on the Outreachy website until the intern announcement deadline on Nov. 16, 2018 at 4pm UTC. Applicants who have not submitted an application will not be able to record or edit their contributions.

**Comment by quinn-codes-synthesis** [comment]
Phew! Proposal posted.

I am a little confused on whether the application needs to be finalized on the 30th or on the 6th -- I had thought the deadline had been postponed, but I'm still seeing some comments (like the one just above) that say the 30th.

Anyway, I welcome feedback so I can polish it in whatever time we have left!

**Comment by sagarpreet-chadha** [comment]
Hi !
The deadline is Nov 6 , 4 PM UTC 🕓 ⏳  .

**Comment by SidharthBansal** [comment]
We have extended it. So that you can work on your proposals well.
:smile:

**Comment by SidharthBansal** [comment]
No more applicants can submit the project. The deadline is over.
